src/sentry/notifications/utils/__init__.py

```diff
@@ -623,6 +630,46 @@ def parallelizable_spans(self) -> List[str]:
     def _find_span_desc_by_id(self, id: str) -> str:
         return get_span_evidence_value(self._find_span_by_id(id))
 
+    @property
+    def duration_impact(self) -> str:
+        transaction_duration = self.transaction_duration
+        time_saved = self._calculate_time_saved()
+        time_saved_percentage = time_saved / transaction_duration * 100
+        return (
+            f"{round(time_saved_percentage,3)}% ({int(time_saved)}ms/{int(transaction_duration)}ms)"
+        )
```

Comment:
I think this'd be better off returning the raw `time_saved` and `transaction_duration` values in `to_dict` and formatting the string in the HTML file. There, you can use filters like `|duration` which will consistently format these durations (and percentages!) and humanize them

Reply:
I've reused the existing duration impact logic in the html template.
I've also move the span evidence key value list (currently only used by my detector), above duration duration impact in the template, so that the span evidence section is in the correct order.

Reply:
Only thing `slow_span_duration` isn't really accurate, any objections to renaming?

Reply:
@DominikB2014 no objections!